opentelemetry-appender-tracing/CHANGELOG.md

```diff
@@ -2,13 +2,14 @@
 
 ## vNext
 
+Fixes [1682](https://github.com/open-telemetry/opentelemetry-rust/issues/1682).
+"spec_unstable_logs_enabled" feature now do not suppress logs for other layers.
+
 ## 0.28.1
 
 Released 2025-Feb-12
 
-- Bump `tracing-opentelemetry` to 0.29
```

Comment:
this is removed, is it intended?

Reply:
yes. we never released with old version!

Reply:
We released - https://crates.io/crates/opentelemetry-appender-tracing/0.28.1

Reply:
I meant - we never released a version which uses tracing-opentelemetry 0.28 or old.  So there is no need to state in changelog that we updated tracing-opentelemetry to 0.29!